src/actions/index.js

```diff
@@ -126,6 +126,10 @@ const postAction = (dataKey, options) => {
     return apiAction('POST', dataKey, options)
 }
 
+const putAction = (dataKey, options) => {
+    return apiAction('PUT', dataKey, options)
+}
```

Comment:
This is un-exported. Will that be a problem? All of its sibling functions are exported near the bottom of this file.